dataland-frontend/src/components/forms/CreateEuTaxonomyForFinancials.vue

```diff
@@ -546,6 +580,42 @@ export default defineComponent({
       this.onThisPageLinks = [...new Set(this.onThisPageLinksStart.concat(this.selectedFinancialServiceOptions))];
     },
 
+    /**
+     * Converts the kpisSection data to a request-friendly object
+     * @param kpiSections Kpi Section data
+     * @returns ObjectType
+     */
+    makeEligibilityKpis(kpiSections: ObjectType) {
+      const eligibilityKpis = Object.keys(kpiSections)
+        .map((key) => {
+          const eligibilityKpi = {
+            key,
+            financialServiceType: euTaxonomyKPIsModel.kpisFieldNameToFinancialServiceType[key] as string,
+          };
+          const section = kpiSections[eligibilityKpi.key] as ObjectType;
+          const field = section[eligibilityKpi.financialServiceType];
+          return { [eligibilityKpi.financialServiceType]: field };
+        })
+        .reduce((all, one) => ({ ...all, ...one }));
+
+      const kpis = Object.keys(kpiSections)
+        .filter((key) => key !== "assetManagementKpis")
+        .map((key) => {
+          const kpi = { [key]: kpiSections[key] };
+          const kpiKey = Object.keys(kpi)[0];
+          if (kpiKey) {
+            const financialServiceType = (euTaxonomyKPIsModel.kpisFieldNameToFinancialServiceType as ObjectType)[
+              kpiKey
+            ];
+            delete (kpi[kpiKey] as ObjectType)[financialServiceType as string];
```

Comment:
I don't get it. There is nothing to delete, I think.

Reply:
Before the delete we'll have an object like this:
```
{
    "creditInstitutionKpis": {
        "CreditInstitution": { ... },
        "tradingPortfolio": { ... },
        "interbankLoans": { ... },
        "tradingPortfolioAndInterbankLoans": { ... },
        "greenAssetRatio": { ... }
    }
}
````
... and we need to delete `CreditInstitution` in order for the request object to be valid.

Reply:
Got it, thanks.